src/driver/expo/ExpoQueryRunner.ts

```diff
@@ -0,0 +1,136 @@
+import {QueryRunnerAlreadyReleasedError} from "../../error/QueryRunnerAlreadyReleasedError";
+import {QueryFailedError} from "../../error/QueryFailedError";
+import {AbstractSqliteQueryRunner} from "../sqlite-abstract/AbstractSqliteQueryRunner";
+import {TransactionAlreadyStartedError} from "../../error/TransactionAlreadyStartedError";
+import {TransactionNotStartedError} from "../../error/TransactionNotStartedError";
+import {ExpoDriver} from "./ExpoDriver";
+import {Broadcaster} from "../../subscriber/Broadcaster";
+
+// Needed to satisfy the Typescript compiler
+interface IResultSet {
+    insertId: number | undefined;
+    rowsAffected: number;
+    rows: {
+        length: number;
+        item: (idx: number) => any;
+        _array: any[];
+    }
+}
+interface ITransaction {
+    executeSql: (
+        sql: string,
+        args: any[] | undefined,
+        ok: (tsx: ITransaction, resultSet: IResultSet) => void,
+        fail: (tsx: ITransaction, err: any) => void
+    ) => void;
+}
+
+/**
+ * Runs queries on a single sqlite database connection.
+ */
+export class ExpoQueryRunner extends AbstractSqliteQueryRunner {
+    
+    /**
+     * Database driver used by connection.
+     */
+    driver: ExpoDriver;
+
+    /**
+     * Database transaction object
+     */
+    private transaction?: ITransaction;
+    
+    // -------------------------------------------------------------------------
+    // Constructor
+    // -------------------------------------------------------------------------
+
+    constructor(driver: ExpoDriver) {
+        super();
+        this.driver = driver;
+        this.connection = driver.connection;
+        this.broadcaster = new Broadcaster(this);
+    }
+
+    /**
+     * Starts transaction.
+     */
+    async startTransaction(): Promise<void> {
```

Comment:
Could you write a comment documenting why you had to override the transaction methods?

Reply:
Yes, that's a good idea. I'll update my PR today.